contracts/0.8.25/vaults/LazyOracle.sol

```diff
@@ -443,7 +443,7 @@ contract LazyOracle is ILazyOracle, AccessControlEnumerableUpgradeable {
         }
 
         // 5. _maxLiabilityShares is greater or equal than _liabilityShares and current `maxLiabilityShares`
```

Comment:
this comment reads weird, should be smth like:
- _maxLiabilityShares must be greater or equal than _liabilityShares
- _maxLiabilityShares must be less or equal than the currently tracked on-chain `record.maxLiabilityShares` (the latter can increase after the ref slot reported)